AI/FuzzyLite/fuzzylite/fl/hedge/Extremely.h

```diff
@@ -31,9 +31,9 @@ namespace fl {
 
     class FL_API Extremely : public Hedge {
     public:
-        std::string name() const;
-        scalar hedge(scalar x) const;
-        Extremely* clone() const;
+        std::string name() const override;
+        scalar hedge(scalar x) const override;
+        Extremely* clone() const override;
 
```

Comment:
Please, do not touch fuzzylite. It is 3rd party library. It had VCMI patches in the past, but now it should be unchanged.